src/pages/TransactionDetails/components/TransactionInfo/TransactionDetailsPanel/TransactionDetailsPanel.tsx

```diff
@@ -0,0 +1,352 @@
+                className='trim-wrapper'
+              >
+                <Trim text={transaction.miniBlockHash} />
+              </NetworkLink>
+              <CopyButton text={transaction.miniBlockHash} />
+            </>
+          ) : (
+            <span>N/A</span>
+          )}
+        </div>
+      </DetailItem>
+
+      <DetailItem title='From'>
+        <div className='d-flex align-items-center'>
+          {addressIsBech32(transaction.sender) ? (
+            <>
+              <AccountLink
+                address={transaction.sender}
+                assets={transaction.senderAssets}
+                hasHighlight
+              />
+              <CopyButton className='me-2' text={transaction.sender} />
+              <ShardLink
+                shard={transaction.senderShard}
+                className='flex-shrink-0'
+                transactionSenderShard
+                hasParanthesis
+              />
+            </>
+          ) : (
+            <ShardSpan shard={transaction.sender} />
+          )}
+        </div>
+      </DetailItem>
+
+      <DetailItem title='To'>
+        <div className='d-flex flex-column'>
+          <div className='d-flex align-items-center'>
+            {isContract(transaction.receiver) && (
+              <span className='me-2 text-neutral-400'>Contract</span>
+            )}
+            <AccountLink
+              address={transaction.receiver}
+              assets={transaction.receiverAssets}
+              hasHighlight
+            />
+            <CopyButton className='me-2' text={transaction.receiver} />
+            {!isNaN(transaction.receiverShard) && (
+              <ShardLink
+                shard={transaction.receiverShard}
+                className='flex-shrink-0'
+                transactionReceiverShard
+                hasParanthesis
+              />
+            )}
+          </div>
+          <div className='d-flex flex-column gap-1'>
+            <TransactionErrorDisplay transaction={transaction} />
+          </div>
+        </div>
+      </DetailItem>
+
+      {receiver !== transaction.receiver && (
+        <DetailItem title='Destination'>
+          <div className='d-flex flex-column'>
+            <div className='d-flex align-items-center'>
+              {isContract(receiver) && (
+                <span className='me-2 text-neutral-400'>Contract</span>
+              )}
+              <AccountLink
+                address={receiver}
+                assets={receiverAssets}
+                hasHighlight
+              />
+              <CopyButton className='me-2' text={receiver} />
+            </div>
+          </div>
+        </DetailItem>
+      )}
+
+      <DetailItem title='Value' className='text-neutral-100'>
+        <FormatAmount
+          value={transaction.value.toString()}
+          showUsdValue={false}
+          showLastNonZeroDecimal
+        />
+        {transaction.price !== undefined && (
```

Comment:
Same comment about the falsey values, applicable everywhere else.

Reply:
updated

Reply:
Awesome. 🔥